alembic/versions/2023_04_13_2048-4f6449741463_initial_migration.py

```diff
@@ -81,6 +81,15 @@ def upgrade():
         sa.Column("name", sqlmodel.sql.sqltypes.AutoString(), nullable=False),
         sa.Column("version", sqlmodel.sql.sqltypes.AutoString(), nullable=False),
         sa.Column("uri", sqlmodel.sql.sqltypes.AutoString(), nullable=True),
+        sa.Column(
+            "dialect",
+            sa.Enum(
+                "SPARK",
+                "TRINO",
```

Comment:
I also saw `POSTGRES` defined in the dialect enum. Does it also need to be included here?

Reply:
Hmm, I'm thinking we should remove postgres, actually, since I'm not sure the way we're generating SQL now will actually be compatible with postgres. Let me do that